prisma/migrations/20250331114534_provider_ip_info/migration.sql

```diff
@@ -0,0 +1,16 @@
+-- CreateTable
+CREATE TABLE "provider_ip_info" (
+    "id" SERIAL NOT NULL,
+    "date" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "provider" TEXT NOT NULL,
+    "lat" TEXT NOT NULL,
+    "long" TEXT NOT NULL,
+    "country" TEXT NOT NULL,
+    "region" TEXT NOT NULL,
+    "city" TEXT NOT NULL,
+
+    CONSTRAINT "provider_ip_info_pkey" PRIMARY KEY ("id")
```

Comment:
does it store IP changes over time? cant we have `provider` as primary key?

Reply:
> does it store IP changes over time? 

Yup.

> cant we have provider as primary key?

We will have multiple rows per provider. New one every week if I remember correctly.
TBH the only reason I added a primary key is because prisma requires one. We could probably switch to `(provider, date)` if we don't like autoincrement id?

Reply:
I vote for `(provider, date)`

Reply:
Done.